hedera-node/test-clients/src/main/java/com/hedera/services/bdd/junit/support/translators/UtilPrngTranslator.java

```diff
@@ -32,7 +33,7 @@ public class UtilPrngTranslator implements TransactionRecordTranslator<SingleTra
 
     @Override
     public SingleTransactionRecord translate(
-            @NotNull SingleTransactionBlockItems transaction, @NotNull StateChanges stateChanges) {
+            @NotNull SingleTransactionBlockItems transaction, @Nullable StateChanges stateChanges) {
```

Comment:
Should use @NonNull above.

Reply:
@iwsimon 
`stateChanges` might come as `null` for certain transactions. This is why we mark them with nullable even in the [BlockStreamTransactionTranslator](https://github.com/hashgraph/hedera-services/blob/stream-manager-spike/hedera-node/test-clients/src/main/java/com/hedera/services/bdd/junit/support/translators/BlockStreamTransactionTranslator.java#L71)

Reply:
I meant using @NonNull (from ackage edu.umd.cs.findbugs.annotations) instead of @NotNull.